jdl/jdl-importer.js

```diff
@@ -183,28 +166,30 @@ function checkForErrors(jdlObject, configuration) {
 }
 
 function importOnlyEntities(jdlObject, configuration) {
-    let { applicationName, applicationType, databaseType, creationTimestamp } = configuration;
+    let { applicationName, applicationType, databaseType } = configuration;
 
-    if (configuration.application) {
-        applicationType = configuration.application['generator-jhipster'].applicationType;
-        applicationName = configuration.application['generator-jhipster'].baseName;
-        databaseType = configuration.application['generator-jhipster'].databaseType;
-        creationTimestamp = configuration.application['generator-jhipster'].creationTimestamp;
+    let application = configuration.application;
+    if (!configuration.application && doesFileExist('.yo-rc.json')) {
+        application = readJSONFile('.yo-rc.json');
+    }
+    if (application && application['generator-jhipster']) {
+        if (applicationType === undefined) applicationType = application['generator-jhipster'].applicationType;
+        if (applicationName === undefined) applicationName = application['generator-jhipster'].baseName;
+        if (databaseType === undefined) databaseType = application['generator-jhipster'].databaseType;
```

Comment:
Please use brackets

Reply:
@MathieuAA I suppose you are asking for curly brackets.
Done